releasenotes/notes/grad_wrapper_expectation-5214882ba482c62f.yaml

```diff
@@ -0,0 +1,3 @@
+---
+fixes:
+  - Include an expectation converter in the `gradient_wrapper` to fix gradient results computed with `qasm` 
```

Comment:
Could you be a bit more precise saying why the results were wrong (missing basis transformations) and replace `qasm` with shot-based simulators (such as `Aer.get_backend('qasm_simulator')`?